endpoints/openrtb2/auction.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/julienschmidt/httprouter"
 	"github.com/mssola/user_agent"
 	"github.com/mxmCherry/openrtb"
+	a "github.com/prebid/prebid-server/analytics"
```

Comment:
Do we really want to rename the analytics module in the code as 'a', would be clearer in the code below if left analytics. Will cause confusion for anyone reading the code at a later date.

Reply:
The word _analytics_ was overused so ran out of ideas 😅 Besides, thought the code looked cleaner this way. I don't mind having it called analytics. Will change it back.